### DISCUSSION TITLE
/fba/inbound/v0/shipments/{shipmentId}/labels does not return UPS label

### DISCUSSION THREAD

**Discussion by njiang** [amzn/selling-partner-api-models#3781] [General]
I'm calling the getLabel api after submitting carton content feed and confirm the transport information.  The returned label only includes FBA barcodes, but doesn't include the UPS label.  I'm wondering if I missed anything in one of those steps? 

Thanks!

**Comment by akowi-mmueller** [comment]
Did you find any held pr solution? I am stuck at a similar problem where I get only FBA labels for DHLEX.
